apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -390,3 +390,47 @@ export const Customization: Story = {
         );
     }
 }
+
+export const TypingUsers: Story = {
+    args: {
+        width: 400,
+        height: 500,
+        typingUsers: [],
+    },
+    argTypes: {
+        typingUsers: {
+            control: 'select',
+            defaultValue: 'No one is typing',
```

Comment:
After loading the story, I would prefer the value 'One user typing' to be selected by default in the select box, and for the demo to reflect this choice.

Reply:
e71c9b02ce630f5a790eacc297e696b421101a5e